Formula/u/urh.rb

```diff
@@ -36,7 +37,8 @@ def install
     EOS
     (libexec/site_packages/"homebrew-libcython.pth").write pth_contents
 
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec, python3)
+    venv.pip_install_and_link(buildpath, build_isolation: false)
```

Comment:
Disabling `build_isolation` to avoid trying to build another `numpy` for build-only usage which currently fails setting up build tools (e.g. `ninja`).

Could switch to `pip install` if we decide to link `libcython`/`cython`. May be worth doing this on Python 3.12 since we follow PEP 668 there; however, this will cause issues for Python 3.10/3.11 users who followed Homebrew's directions to manually run `pip3[.10|.11] install cython` (as we cannot unlink user `pip` installed packages).

Reply:
maybe good to add this linkage as code comment besides PR comment so that we can pick it up later.